packages/desktop-client/src/components/reports/Header.js

```diff
@@ -129,20 +147,113 @@ function Header({ title, start, end, show1Month, allMonths, onChangeDates }) {
         <Button
           bare
           style={{ marginLeft: 15 }}
-          onClick={() => onChangeDates(...getLatestRange(12))}
+          onClick={() => onChangeDates(...getLatestRange(11))}
         >
           1 Year
         </Button>
         <Button
           bare
-          style={{ marginLeft: 15 }}
+          style={{
+            marginLeft: 15,
+            display: showAllTime ? 'inherit' : 'none',
+          }}
           onClick={() => onChangeDates(...getFullRange(allMonths))}
         >
           All Time
         </Button>
       </View>
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'flex-start',
+          paddingTop: 15,
+        }}
+      >
+        <View style={{ marginRight: 10, marginTop: 2, marginBottom: 5 }}>
+          <FilterButton onApply={onApplyFilter} disableFilter={disableFilter} />
+        </View>
+
+        {filters && filters.length > 0 && (
+          <AppliedFilters
+            filters={filters}
+            onUpdate={onUpdateFilter}
+            onDelete={onDeleteFilter}
+          />
+        )}
+      </View>
```

Comment:
Since this is disabled for most of the reports right now, do you think it would make more sense to include it in the report body? For income & expense I could see a layout like this:

> <kbd>Expense ▾</kbd> 🎚️ Filter <kbd>**payee** is **foo** ×</kbd>
>
> 🥧 Totals &nbsp; &nbsp; 📊 Trends